src/Command/UpdateCommand.php

```diff
@@ -94,7 +91,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
 
             //get versions as [version => url]
-            $versions = $info['versions'] ?: [];
```

Comment:
Yep, Should probably go into another PR. I had to rebuild the database completely, and I was getting a notice for each and every package, so I had to fix it.